sql/informix-sql/InformixSQLParser.g4

```diff
@@ -72,6 +75,33 @@ dropView
     : DROP VIEW (IF EXISTS)? viewName (CASCADE | RESTRICT)?
     ;
 
+//https://www.ibm.com/docs/en/informix-servers/14.10?topic=statements-drop-access-method-statement
+dropAccessMethod
+    : DROP ACCESS_METHOD (IF EXISTS)? accessMethodName RESTRICT
+    ;
+
+//https://www.ibm.com/docs/en/informix-servers/14.10?topic=statements-drop-aggregate-statement
+dropAggregate
+    : DROP AGGREGATE (IF EXISTS)? aggregateName
+    ;
+
+//https://www.ibm.com/docs/en/informix-servers/14.10?topic=statements-drop-database-statement
+dropDatabase
+    : DROP DATABASE (IF EXISTS)? databaseName
+    ;
+
+databaseName
+    : identifier
+    ;
+
+aggregateName
+    : identifier
+    ;
+
+accessMethodName
+    : identifier
+    ;
```

Comment:
Do we really need separated rules for actually plain identifiers? Can't we use just `identifier` in `dropAccessMethod`, `dropAggregate`, `dropDatabase`?

Reply:
Use different names for every statements,   grammar prompts will be easier,  in the future.

Reply:
> Use different names for every statements, grammar prompts will be easier, in the future.

Not sure because they are used only in one use-site and there are no other identifiers that could ambiguate.

Also, what about using [rule element labels](https://github.com/antlr/antlr4/blob/master/doc/parser-rules.md#rule-element-labels) instead of full-fledged rules? It looks more reasonable and faster.

```g4
dropAccessMethod
    : DROP ACCESS_METHOD (IF EXISTS)? accessMethodName=identifier RESTRICT
    ;
```

Reply:
Yeah, using rule element labels is the best way. I'll rewrite them。Thank you, @KvanTTT .